File: [프로그래머스] 코딩테스트 입문_소인수분해.js
function solution(n) {
    var answer = [];
    let i = 2;
    while(i <= n){
        if(n % i == 0){
            n /= i
            if(!answer.includes(i)){
                answer.push(i);
            }            
        } else {
            i++;
        }
    }
    return answer;
}

File: [프로그래머스] 코딩테스트 입문_소인수분해.py
def solution(n):
    answer = []
    result = []
    i = 2
    while i <= n:
        if n % i == 0:
            n //= i
            answer.append(i)
        else:
            i += 1
    for i in answer:
        if i not in result:
            result.append(i)
    return result
--------------------------------------
def solution(n):
    answer = set()
    i = 2
    while i <= n:
        if n % i == 0:
            n //= i
            answer.add(i)
        else:
            i += 1
    return sorted(list(anwer))
